.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+# We are supposed to use architect-orb to build&push container images.
+# However, architect-orb has strict rules and tests. We don't have full control over this repository.
+# That is why we introduced this custom configuration in this repository.
+# Keep this file as it is while merging changes from upstream repository.
+
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: ubuntu-2204:2022.04.2
+    steps:
+      - checkout
+      - run:
+          name : Build container
+          command: |
+            export IMG=capvcd:latest
```

Comment:
```suggestion
            IMG_TAG=$(git rev-parse --short HEAD)
            export IMG=capvcd:${IMG_TAG}
```

Reply:
What is the benefit of this change? We will never push `capvcd:latest`. It is just a temporary name.